### PR TITLE
Issues/215 topics page tests

### PR DESCRIPTION
**Pull request by Mgrdich**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [ ] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by Mgrdich** [comment]
I left out filters cause i think it should be done in separate issue , cause that component is way to big has to be divided into sub component and those  should be mocked in Filters.

**Comment by Mgrdich** [comment]
it can me merged to the master before #1880, since it contains one migration that is needed for that task

**Review by lazzy-panda — APPROVED** [review]

File: kafka-ui-react-app/src/components/Topics/Topic/SendMessage/__test__/SendMessage.spec.tsx [hunk 1]
```diff
@@ -30,22 +35,41 @@ jest.mock('json-schema-faker', () => ({
...
+      </Router>
+      <S.AlertsContainer role="toolbar">
+        <Alerts />
+      </S.AlertsContainer>
+    </>,
     { store }
   );
 };
 
+const RenderAndSubmitData = async (error: string[] = []) => {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/SendMessage/__test__/SendMessage.spec.tsx:63]: Why does the name of this method begin with a capital letter?

File: kafka-ui-react-app/src/components/Topics/Topic/__tests__/Topic.spec.tsx [hunk 2]
```diff
@@ -9,64 +9,78 @@ import {
   clusterTopicSendMessagePath,
 } from 'lib/paths';
 
+const TopicText = {
```
Comment (workshur) [inline kafka-ui-react-app/src/components/Topics/Topic/__tests__/Topic.spec.tsx:12]:
```suggestion
const topicText = {
```

or 
```suggestion
const TOPIC_TEXT = {
```

**Review by workshur — APPROVED** [review]
Looks good to me

**Pull request merged**